code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -365,7 +365,6 @@ func (sc *StorageSmartContract) newAllocationRequestInternal(
 		return allocatedBlobbers[i].ID < allocatedBlobbers[j].ID
 	})
 
-	sa.Blobbers = allocatedBlobbers
```

Comment:
allocatedBlobbers does not seem to be used anywhere, so we should be able to remove it.